src/software/thunderscope/robot_communication.py

```diff
@@ -23,26 +25,27 @@ def __init__(
         current_proto_unix_io: ProtoUnixIO,
         multicast_channel: str,
         interface: str,
-        disable_estop: bool,
-        estop_path: os.PathLike,
+        estop_mode: EstopMode,
+        estop_path: os.PathLike = None,
         estop_buadrate: int = 115200,
```

Comment:
nit:
```suggestion
        estop_baudrate: int = 115200,
```
The change in the doc can also be reverted